internal/pkg/controllers/spod/setup.go

```diff
@@ -43,8 +44,15 @@ type DaemonTunables struct {
 	WatchNamespace      string
 }
 
+var operatorNamespace string
```

Comment:
Do we want to be on the safe side of life and protect this global var with a mutex?